telcell/models/simplemodel.py

```diff
@@ -0,0 +1,76 @@
+import datetime
+import numpy as np
+from typing import List
+
+from telcell.data.models import Measurement, Track, MeasurementPair
+
+
+def get_measurement_with_minimum_time_difference(track: Track,
+                                                 timestamp: datetime) \
+        -> Measurement:
+    """
+    Finds the measurement in the track with the smallest time
+    difference compared to the timestamp.
+
+    @param track: A history of measurements for a single device.
+    @param timestamp: The timestamp used to find the closest measurement.
+    @return: The measurement that is the closest to the timestamp
+    """
+    track_a_times = [x.timestamp for x in track.measurements]
+    time_difference_with_timestamp = [abs(timestamp - x)
+                                      for x in track_a_times]
+    index = np.argmin(time_difference_with_timestamp)
+    return track.measurements[index]
```

Comment:
Zou deze hele functie niet in een oneliner kunnen: `return min(measurements, key=lambda m: abs(m.timestamp - timestamp))

Reply:
Mooie! die werkt ook idd, heb hem aangepast